README.md

```diff
@@ -134,7 +134,7 @@ Each child module in turn relies by default on the included [`cppbuild.sh` scrip
  * NumPy 1.19.x  https://github.com/numpy/numpy
  * SciPy 1.5.x  https://github.com/scipy/scipy
  * Gym 0.17.x  https://github.com/openai/gym
- * LLVM 10.0.x  http://llvm.org/releases/download.html
+ * LLVM 11.0.x  http://llvm.org/releases/download.html

```

Comment:
I suppose I could change this to 11.0.0 as this would be more precise, but I left it as is